client/components/common/SelectElement/index.vue

```diff
@@ -2,9 +2,26 @@
   <tailor-dialog
     @click:outside="close"
     :value="true"
-    header-icon="mdi-transit-connection-variant"
+    :header-icon="headerIcon"
+    scrollable
     width="650">
-    <template v-slot:header>{{ heading }}</template>
+    <template v-slot:header>
+      {{ heading }}
+      <v-btn
```

Comment:
Add slot instead and expose toggleSelectAll and allElementsSelected (creates error for element link as well).

Reply:
Seems that creating a slot just moves the problem to the consumers and requires repeatable code in them (for similar behaviour among them, of course). Instead, I've managed to handle toggling ability inside the component by checking whether it allows selecting multiple items and the content is loaded or not. Let me know if you see any drawbacks in this approach 😉.